src/cmake/FindQhull.cmake

```diff
@@ -4,7 +4,7 @@
 # Specify components:
 #
 # ::
-#   qhull          = Deprecated interface: use imported target Qhull::qhull
+#   libqhull       = Deprecated interface: use imported target Qhull::libqhull
```

Comment:
this line is a bit redundant now? or rather, probably needs to be rephrased

Reply:
No, its correct. Qhull::libqhull is deprecated as it is not reentrant.